common/rya.api/src/main/java/org/apache/rya/api/client/CreatePCJ.java

```diff
@@ -27,8 +29,28 @@
 @DefaultAnnotation(NonNull.class)
 public interface CreatePCJ {
 
+    public static enum QueryType{Construct, Projection, Periodic};
+    public static enum ExportStrategy{Rya, Kafka, NoOpExport};
```

Comment:
Add javadocs for the enums and their types.  Also change the names for the types to all caps with words separated by _.